src/Gui/TaskView/TaskView.cpp

```diff
@@ -751,18 +748,11 @@ void TaskView::addTaskWatcher()
         taskPanel->addStretch();
     updateWatcher();
 
-#if QT_VERSION >= QT_VERSION_CHECK(5, 12, 0)
-    // Workaround to avoid a crash in Qt. See also
-    // https://forum.freecad.org/viewtopic.php?f=8&t=39187
-    //
-    // Notify the button box about a style change so that it can
-    // safely delete the style animation of its push buttons.

```

Comment:
Some comments are really important. This is such a thing as we would never be able to find and remove this workarounds without it.

Reply:
This is a comment regarding a bug in Qt version 5.12

Reply:
But you have only removed the comment about what the workaround was for, not the workaround itself.

If we still need the code, then I think we need the comment about the issue